dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facilities.repo.ts

```diff
@@ -263,4 +263,15 @@ export class TfmFacilitiesRepo {
       'facilitySnapshot.ukefFacilityId': { $eq: ukefFacilityId },
     });
   }
+
+  /**
+   * Checks whether or not a document exists which has the matching UKEF facility ID
+   * @param ukefFacilityId - The UKEF facility ID
+   * @returns Whether a not a document with that id exists
```

Comment:
```suggestion
   * @returns Boolean
```

Reply:
This is a typescript file so we should not be writing types in the docstrings. (It also returns `Promise<boolean>` not `Boolean` as can be seen from the function signature